### ISSUE TITLE
Greek text added.

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/INM#4]
The 10000+ greek text fragments of INM have now been added to csl-orig/v02/inm/inm.txt.
This issue devoted to the details.

**Comment by Andhrabharati** [comment]
Just happened to see this difference-

Jim's version:
```
<L>164<pc>013-1<k1>Adideva<k2>Adideva<h>3
{@Ādideva@}<sup>2</sup>¦ = Kṛṣṇa: XIII, 6893, 7373.
```

AB version:
```
<L>164<pc>013-1<k1>आदिदेव<k2>आदिदेव<h>3
{@Ādideva@}³¦ = Kṛṣṇa: XIII, 6893, 7373.
```

Only the metaline homonym `<h>` portion is updated (from 2 to 3), but not the HW `<sup>` portion; probably it would be done in the next step.

**Comment by funderburkjim** [comment]
Yes that sup-2 is a typo. Should be sup-3.  Above commit corrects.

**Comment by funderburkjim** [comment]
## dev-details

The [greek](https://github.com/sanskrit-lexicon/INM/tree/main/greek) folder contains essential code and a detailed readme
of the process by which Greek text was inserted into csl-orig/v02/inm/inm.txt at commit  9fc6ed8.

The basic idea was to align the empty lang tags in the csl version (A) with the greek text in Andhrabharati's 'consolidated'
version [ΙΝΜ-devanagari_consolidated (Andhrabharati).txt](https://github.com/sanskrit-lexicon/csl-devanagari/files/7632615/-devanagari_consolidated.Andhrabharati.txt).

Along the way, several changes to headwords were made in the A version, and a few changes to the B version.
The final changed 'B' version is [inm_slp1_L2_02.txt](https://raw.githubusercontent.com/sanskrit-lexicon/INM/main/greek/inm_slp1_L2_02.txt).
This also converted back to devanagari and available as
[inm_deva_L2_02.zip](https://github.com/sanskrit-lexicon/INM/files/7696354/inm_deva_L2_02.zip)

One 'temporary' detail is the presence of several `<gx>X</gx>` items in the L2 version.  These mark Greek texts originating
from the additions-corrections section of INM.  Since I was not aiming to include those in the present work, it was 
necessary to identify them for alignment of other greek texts.

**Comment by Andhrabharati** [comment]
This indeed has turned out to be a good 'team-work'.

**Seen that the majority of corrections/updates are in the meta-line markings.**

@funderburkjim 
How about changing the "Do.ⁿ¦" HWs to the proper words from the prev. entries?
I had thought of doing it myself (in my working plan initially), but somehow it did not happen!

Also some entries need to be marked with `<h>n` in meta-line and/or the HW to be made with a `<sup>n</sup>`, as the subsequent (split) entries have the `<h>2` etc.
[I had done this exercise in few entries initially, but looks many are left!]

**Comment by Andhrabharati** [comment]
> Since I was not aiming to include those in the present work, it was
> necessary to identify them for alignment of other greek texts.

Not doing it in the present **phase** [why not complete it in one spree?], or just in the present **step**?

**Comment by Andhrabharati** [comment]
> a detailed readme
> of the process by which Greek text was inserted into csl-orig/v02/inm/inm.txt

@funderburkjim 

Went through the readme file, and noted how much **extra work** is given to you, with my style of working.

But hope you would've enjoyed doing it all, it being a worth-while exercise, to put things back in the cdsl style.

Do you wish to suggest any changes in my working (in future)?

**Comment by Andhrabharati** [comment]
> NOTE: page 474 of pdf is poor quality [A note in the readme file]

Here is a better page to use (from another scan version)-
[INM_474.pdf](https://github.com/sanskrit-lexicon/INM/files/7696566/INM_474.pdf)

**Comment by funderburkjim** [comment]
That pdf page is so much better!   Would you share a link to a pdf of the entire work ?

**Comment by Andhrabharati** [comment]
Here is the link-
https://archive.org/download/in.ernet.dli.2015.98210/2015.98210.An-Index-To-The-Names-In-The-Mahabharata.pdf

**Comment by funderburkjim** [comment]
Noticed that the [single greek word](https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-968273563)  was missing the initial accented-capital-letter Alpha in cologne version.  Now corrected.

**Comment by funderburkjim** [comment]
Also marked that word as italic, per [comment](https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-968314726) by @jmigliori .

**Comment by funderburkjim** [comment]
Comparison between page 474 from the full download link above (left) and
the separate page 474 from above (right):

![image](https://user-images.githubusercontent.com/6393033/145694197-7f99e47c-bfe4-4ac4-b057-0d78fc3562a2.png)

Based on this comparison, the individual page 474 comes from a different and better source than the full archive org pdf.

**Comment by Andhrabharati** [comment]
Sorry @funderburkjim, for my error in giving the link.

The source you wanted is from the GRETIL e-library, not from the archive.org.
http://gretil.sub.uni-goettingen.de/gretil_elib/Sor904__Sorensen_IndexMahabharata.pdf

**Comment by Andhrabharati** [comment]
> Noticed that the [single greek word](https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-968273563) was missing the initial accented-capital-letter Alpha in cologne version. Now corrected.

> Also marked that word as italic, per [comment](https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-968314726) by @jmigliori .
> 

@funderburkjim,

Like to bring to your notice that  **all the Greek letters throughout are to be in italics**, not just this single word.

Pl. refer to my earlier post-
https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-957194357

I thought this could be taken care by a script while displaying, hence did not mark them in the file thus (as italics).

**Comment by gasyoun** [comment]
> Like to bring to your notice that all the Greek letters throughout are to be in italics, not just this single word.

A small detail.

**Comment by funderburkjim** [comment]
> Greek letters are in italics when after a verse number or adhyAya number

This comment seems to qualify which Greek letters are to be in italics.
These questions:
1. What is an example of Greek letter after a verse number
2. What is an example of Greek letter after adhyAya number?
3. Are there any Greek text instances that should not be in italics?

**Comment by funderburkjim** [comment]
Thanks for ref of Gretil pdf.  I've downloaded it looks good.

Perhaps we should entirely replace our INM page links to use the pages of this pdf.  

However, I would like to see the copyright limitations to the pdf, if any.   But the 'parent' link to the pdf is not accessible, and the toplevel url doesn't mention pdfs.

**Comment by Andhrabharati** [comment]
is this the parent link you are looking for-

http://opac.sub.uni-goettingen.de/DB=1.20/LNG=EN/SID=09f7d727-0/CMD?ACT=SRCHA&IKT=1016&SRT=YOP&TRM=index+to+names+in+mahabharata

**Comment by Andhrabharati** [comment]
> > Greek letters are in italics when after a verse number or adhyAya number
> 
> This comment seems to qualify which Greek letters are to be in italics. These questions:
> 
>     1. What is an example of Greek letter after a verse number
The verse no. is the number in 'normal'.
![image](https://user-images.githubusercontent.com/75209130/145741908-697f21eb-896f-4e15-9bcb-523e3c5e6e7e.png)

> 
>     2. What is an example of Greek letter after adhyAya number?

The adyAya no. is the number in 'bold'.

![image](https://user-images.githubusercontent.com/75209130/145741813-1eebec5a-fa8d-4f5e-afbd-519c40576b88.png)

> 
>     3. Are there any Greek text instances that should not be in italics?

None, after changing the `Ἀ` at `<L>93`

You may also see this, in the context-
https://github.com/sanskrit-lexicon/csl-devanagari/issues/34#issuecomment-991842443

4. There are also places where the Greek letters are not **attached** to a number, but are independently placed (either open or in braces)

> —§ 584 (do.): VI, {@81@}α, 3533, α¹, 3557; {@86@}`<sup>g</sup>`, 3823.

**In summary, all the Greek occurrences in the text are in italics.**

**Comment by funderburkjim** [comment]
One other difference noted regarding the Greek after (bold) adhyAya number:  In cologne version, these Greek letters are also bold.  In your version, I believe, normally not bold.

Examination of a few cases in the scan suggests generally your version is correct.

So, I think the Cologne version should also remove bold markup from such cases. 
For instance:
```
OLD:
VI, {@9<lang n="greek">ν</lang>,@} 372
NEW:
VI, {@9@}<lang n="greek">ν</lang>, 372
```
Agree?

**Comment by Andhrabharati** [comment]
yes, pl.go ahead.

**Comment by funderburkjim** [comment]
## Greek text italic.
As you expected, the italic markup could be added in the display program, and that is the way it was done.
The change is in the basicadjust.php program, which occurs both in csl-apidev and csl-websanlexicon repositories which have code for (a) simple-search and (b) basic, list, etc. displays, respectively.

**Comment by funderburkjim** [comment]
@Andhrabharati   I think I've done all that is needed regarding Greek text in INM, as mentioned above.  Have you noticed anything I overlooked?

**Comment by Andhrabharati** [comment]
haven't seen you mentioning about the 'bold' alpha.

I would suggest that it should also be made normal, like all other Greek letters; and that's all on Greek text in INM.

**Comment by funderburkjim** [comment]
Current status is that all Greek text is normal, including 'alpha'.
Example:
```
<L>859<pc>092-1<k1>AryA<k2>AryA<h>1
{@Āryā@}<sup>1</sup>.¦ § 500 (Skandop.): III, {@228@}<lang n="greek">α</lang>, 14396 (a Mātṛ, one
```
See [simple-search display](https://sanskrit-lexicon.uni-koeln.de/simple/inm/AryA/slp1).

No problem there that I see.

**Comment by Andhrabharati** [comment]
yes, the alpha is normal there.

**Comment by Andhrabharati** [comment]
now, this issue is 'done'.

**Comment by gasyoun** [comment]
> now, this issue is 'done'.

Close issue?

**Comment by funderburkjim** [comment]
About 50 print changes in the spelling of headwords were made in the early steps of aligning the Cologne and Andhrabharati versions of inm.  These now noted in the print change file for inm in csl-corrections repository. See commit above.
